collect_app/src/test/java/org/odk/collect/android/utilities/FormNameUtilsTest.java

```diff
@@ -40,4 +40,21 @@ public void normalizeFormNameTest() {
         assertEquals("Lorem  ipsum", normalizeFormName("Lorem\n\nipsum", true));
         assertEquals(" Lorem ipsum ", normalizeFormName("\nLorem\nipsum\n", true));
     }
+
+    @Test
+    public void formatFilenameFromFormNameTest() {
+        assertNull(FormNameUtils.formatFilenameFromFormName(null));
```

Comment:
I've typically favored hamcrest `assertThat` with matchers so assertions read more like sentences and I believe @seadowg and @grzesiek2010 typically do the same:
- `assertThat(FormNameUtils.formatFilenameFromFormName(null), is(nullValue()));`
- `assertThat(FormNameUtils.formatFilenameFromFormName("simple"), is("simple"));`

I don't feel strongly about imposing a standard here because these aren't particularly hard to read but at least wanted to make sure you knew about those tools.

Reply:
Thanks for the guidance. I initially tripped myself up on the JUnit 'assertThat'. Switching to hamcrest gave me what I need.

Reply:
Aha! Ok, I think this might be worth a note in the contributor guide. I forgot there was that ambiguity.

Reply:
I can add that note, but it seems more appropriate for the Coding Guidelines doc.

Reply:
Good point! It just got split out so I forgot. 😊

Reply:
Apparently I forget a lot of things...

Reply:
That's why we document them, so we can forget!